src/domaindiag.F

```diff
@@ -10102,39 +10102,26 @@ subroutine momentum_flux(u3d,v3d,w3d,t13,t23,m13,m23,rf,rf0,c1,c2,dum3,dum4,dum5
 
   !----------------------
 
-    
-#ifndef _B4B21R
-    !$acc parallel default(present) private(i,j,k) reduction(+:vbart) reduction(+:vfrt,vfdt)
+    !$acc parallel default(present)
```

Comment:
@sjsprecious, you should be aware that the momentum_flux() subroutine is not currently called in CM1.  I had commented out its call because it cause an OpenACC error.  Unless you have already reactivated the code, I would encourage you to turn it back on to verify that it now works with your changes.